site/docs/numbers/manageInventory.mdx

```diff
@@ -0,0 +1,1203 @@
+---
+id: manageInventory
+title: How to manage your inventory
+slug: /numbers/guides/searchNumbers
```

Comment:
I would change the slug here to /numbers/guides/manageInventory unless it needs to be /numbers/guides/searchNumbers

Reply:
We will need to setup a redirect for the URL so just ping me when this is going live